style.css

```diff
@@ -0,0 +1,233 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito&display=swap');
+
+*{
+  margin: 0;
+  padding: 0;
+
+  font-size: 62.5%;
+
+  box-sizing: border-box;
+}
+
+body{
+  height: 100vh;
```

Comment:
What would you suggest, is it better to use a fixed height here, or a min-height ?

Reply:
I personally prefer this to make my content fixed to the 1 page only